xcell/mappers/mapper_DELS.py

```diff
@@ -186,8 +192,9 @@ def _get_comp_map(self):
 
     def _get_binary_mask(self):
         if self.bmask is None:
-            self.bmask = hp.ud_grade(hp.read_map(self.config['binary_mask']),
-                                     nside_out=self.nside)
+            bmsk = hp.read_map(self.config['binary_mask'])
+            bmsk = rotate_mask(bmsk, self.rot)
+            self.bmask = hp.ud_grade(bmsk, nside_out=self.nside)
```

Comment:
You need to binarize this mask.

Reply:
Also missing from this class:
- Rotate the star map
- Rotate the completeness map

Reply:
Depends on the case. If you binarize before udgrading, when you downgrade you'll get a non-binary mask. If upgrading (which is the most likely scenario for our final run), it doesn't really matter, but it doesn't hurt to be careful.